ee/clickhouse/views/test/test_clickhouse_trends.py

```diff
@@ -332,3 +320,67 @@ def test_insight_trends_compare(self):
         assert sorted([p["id"] for p in prev_people]) == sorted(
             [str(created_people["p1"].uuid), str(created_people["p2"].uuid)]
         )
+
+
+class ClickhouseTestTrendsGroups(ClickhouseTestMixin, LicensedTestMixin, APIBaseTest):
```

Comment:
Moved test here because i changed it to an API test. Also, added a different class just so it's clearer and we can move it to a different file just for organizations sake